ttt/board.py

```diff
@@ -11,12 +11,20 @@ def place_marker(self, space, marker):
     def is_full(self):
         return '-' not in self._spaces
 
-    def is_available_space(self, space):
-        return False if self._is_out_of_bounds_space(space) or self._is_occupied_space(space) else True
+    def is_valid_space(self, space):
+        return False if self._is_out_of_bounds_space(space) or \
```

Comment:
Try to avoid statements that return a boolean if some boolean condition is met. Instead, just return the result of the boolean expression. The entire statement will usually be easier to read if you name the methods in such a way that they read in the truthy case, e.g.:

```
return self._is_space_in_bounds(space) and self._is_space_unoccupied(space)
```